packages/tychon/data_stream/arp/elasticsearch/ingest_pipeline/default.yml

```
@@ -0,0 +1,31 @@
+---
+description: Pipeline for TYCHON ARP Tables
+processors:
+
+  - pipeline:
+      tag: pipeline
```

```suggestion
      tag: pipeline_common_init
```
?
If so, similar below.

---

Yep, that's good. Done.